code/src/components/LikeButton.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+
+
+export const LikeButton = (props) => {  
+  return (    
+      <button className={`like-button ${props.thoughts.hearts === 0 ? "not-liked" : "liked"}`} onClick= {() => props.OnLikesIncrease(props.thoughts._id)}>
+        <span className="heart-on-button-icon" role="img" aria-label="Heart">💗</span>
```

Comment:
not 100% sure but think one can use props destructuring as well as an alternative ?  Not sure if it only works when passing single props ? For example: 

```
export const ComponentName = ({ thoughts,  hearts }) => {
  //
}
```